src/Rules/Resources/HowToFix.resx

```diff
@@ -523,7 +523,7 @@ Where appropriate, use a standard localized landmark type:
  · Use "navigation" for an area containing links for page or site navigation.
  · Use "search" for an area of the page containing search functionality.
 If none of the standard landmark types is applicable, provide a string that concisely describes its content.</value>
-    <comment>Brief guidance on how to fix an accessibility problem. {Locked="LocalizedLandmarkType"}</comment>
+    <comment>Brief guidance on how to fix an accessibility problem. {Locked="banner", "complementary", "contentinfo", "form", "LocalizedLandmarkType", "navigation", "search"}</comment>
```

Comment:
The term "main" appears in line 522. Should it also be in the list of locked strings?